formulas/wick-base/functions/wick-is-apt-available

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Checks whether Apt and Dpkg type applications are running.
+#
+# Examples
+#
+#   wickIsAptAvailable
+#
+# Returns 0 when no Apt or Dpkg type applications are found running.
+# Returns >0 when an Apt or Dpkg type application is found running.
```

Comment:
These sentences are inconsistent, one is plural and one is singular, was that intentional? There is also a missing space after the operator - `Returns >0` vs. `Returns > 0`, which seems pointless to bring up but for some reason it reads weird to me without the space. Ignore this if that doesn't make sense to you.